service/replication.go

```diff
@@ -112,8 +112,16 @@ func (s *service) CreateRemoteVolume(ctx context.Context,
 	if err != nil {
 		return nil, status.Errorf(codes.Internal, "failed to sync data %s", err.Error())
 	}
-	volumeSize := isiConfig.isiSvc.GetVolumeSize(ctx, isiPath, volName)
-	log.Info("Volume size got: ", volumeSize)
+	// Get source quota size if assigned and set the same on target
+	var volumeSize int64
+	sourceQuota, err := isiConfig.isiSvc.client.GetQuotaWithPath(ctx, exportPath)
+	if err != nil && !strings.Contains(err.Error(), "not found:") {
```

Comment:
Is the error coming from the array API or our code? Seems fragile if the API returns a different error. Hopefully they do not change this often.

Reply:
This error is from our code in gopowerscale library. Array API just returns empty list if no quota is set for that path.